keps/4803-workload-level-wait-for-pods-ready/README.md

```
@@ -0,0 +1,237 @@
+    - [Workload Spec](#workload-spec)
+    - [Resource Annotations](#resource-annotations)
+  - [Controller](#controller)
+    - [Workload](#workload)
+    - [Jobs / Jobframework](#jobs--jobframework)
+  - [Webhooks](#webhooks)
+    - [Workload](#workload-1)
+    - [Managed resources (Jobs, Deployments, StatefulSets, etc.)](#managed-resources-jobs-deployments-statefulsets-etc)
+  - [Test Plan](#test-plan)
+    - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Integration tests](#integration-tests)
+      - [controller/core/workload](#controllernetworkworkload)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)